db_test.go

```diff
@@ -129,3 +129,28 @@ func TestDbNoForwardingHistory(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, fwds, 0)
 }
+
+func TestForwadingHistoryDelete(t *testing.T) {
+	// Create a db that will store HTLCs.
+	ctx := context.Background()
+	db, cleanup := setupTestDb(t, 5)
+	defer cleanup()
+
+	// Write a test HTLC and assert that it's stored.
+	htlc := testHtlc(1)
+	require.NoError(t, db.RecordHtlcResolution(ctx, htlc))
+
+	fwds, err := db.ListForwardingHistory(ctx, time.Time{}, time.Unix(1000000, 0))
+	require.NoError(t, err)
+	require.Len(t, fwds, 1)
+
+	// Modify the db to have a zero limit on forwarding history. We don't recreate
+	// the test db because it would re-create the file. Run limitHTLCRecords once
```

Comment:
Maybe not the most beautiful way to cover, but good enough I think.

Reply:
Yeah :| didn't think it was worth updating the test db setup for this one case, but I agree.. ugly